src/main/webapp/jsp/instructorFeedbackResponseCommentsLoad.jsp

```diff
@@ -4,6 +4,7 @@
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 
 <div class="hidden number-of-pending-comments">${data.numberOfPendingComments}</div>
+
```

Comment:
Remove this file from the PR if there are no changes that are supposed to be done to it.